python/nav/web/navlets/graph.py

```diff
@@ -56,7 +57,9 @@ def get_context_data_view(self, context):
         self.title = self.get_title()
         show_controls = self.preferences.get('show_controls')
         context['hide_buttons'] = 'false' if show_controls else 'true'
-        context['graph_url'] = self.preferences.get('url')
+        url = self.preferences.get('url')
+        timestamp = int(datetime.now().timestamp())
+        context['graph_url'] = f'{url}&bust={timestamp}'
```

Comment:
Cache-busting? Add a comment about it.

Reply:
The bust parameter was defined here. 
https://github.com/Uninett/nav/blob/16d641956c62877edc53e0a28eabe07842c5a871/python/nav/web/static/js/src/plugins/navlet_controller.js#L129-L139

I created a new staticmethod on the Navlet class, `add_bust_param_to_url`